python/tvm/topi/nn/softmax.py

```diff
@@ -136,16 +136,38 @@ def log_softmax(x, axis=-1):
     output : tvm.te.Tensor
         2-D output with same shape
```

Comment:
nit: change this to "N-D" as well

Reply:
@shingjan Can you follow up this in your PR?

Reply:
will do! Thanks

Reply:
@masahi This typo is fixed in #11978